sda/cmd/sda.md

```diff
@@ -0,0 +1,21 @@
+sda - sensitive-data-archive
+============
+
+Repository:
+[neicnordic/sensitive-data-archive](https://github.com/neicnordic/sensitive-data-archive)
+
+`sda` repository consists of a suite of services which are part of [NeIC Sensitive Data Archive](https://neic-sda.readthedocs.io/en/latest/) and implements the components required for data submission.
+It can be used as part of a [Federated EGA](https://ega-archive.org/federated) or as an isolated Sensitive Data Archive.
+`sda` was built with support for both S3 and POSIX storage.
+
+The SDA submission pipeline has four main steps:
+
+1. [Ingest](ingest.md) splits file headers from files, moving the header to the database and the file content to the archive storage.
+2. [Verify](verify.md) verifies that the header is encrypted with the correct key, and that the checksums match the user-provided checksums.
+3. [Finalize](finalize.md) associates a stable accessionID with each archive file and backups the file.
+4. [Mapper](mapper.md) maps file accessionIDs to a datasetID.
+
+There are also three additional support services:
+
+1. [Intercept](intercept.md) relays messages from Central EGA to the system.
+3. [s3inbox](s3inbox.md) proxies uploads to the an S3 compatible storage backend.
```

Comment:
I assume the third here is `sda-auth` although it is not yet part of the unified code.

Reply:
ah sry numbering, will fix it, yes, we can add that when we make it part of the unified code.
the idea is to add to that as it is part of unified code